### DISCUSSION TITLE
2023년 10월 08일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#86] [General]
# 2023년 10월 08일

TODO 미션 3단계 제출  [priority:: highest]  [created:: 2023-10-05]  [completion:: 2023-10-08] 글쓰기 미션  [priority:: medium]  [due:: 2023-10-30] 링글 예습  [priority:: highest]  [due:: 2023-10-08] 아코와 페어프로그래밍  [p…

https://vero.wiki/2023-10-08/

**Comment by xxeol2** [comment]
DeleteChangeList하면 됩니다
저도 며칠전에 3달간 상주하던 푸우 없앴습니다

**Reply by Cyma-s** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅌㅋㅋㅋㅋㅋ
